config/speedtest.php

```diff
@@ -40,4 +40,15 @@
      */
     'skip_ips' => env('SPEEDTEST_SKIP_IPS', ''),
 
+    /**
+     * Threshold settings.
+     */
+
+     'threshold_enabled' => env('THRESHOLD_ENABLED', false),
+
+     'threshold_download' => env('THRESHOLD_DOWNLOAD', '0'),
```

Comment:
Instead of passing a string as `'0'` you'll want to pass it as an integer `0` to keep the type the same.

Reply:
Whoepsie, change it.